config/ECS/otel-agent-ecs-default-config.yaml

```diff
@@ -0,0 +1,37 @@
+extensions:
+  health_check:
```

Comment:
Commenting at the top since I can't comment on the file name itself: `ECS` is capitalized in the file path for this file and the one above...

Reply:
Wrong file committed.